core/admin/mailu/models.py

```diff
@@ -285,9 +294,9 @@ def dns_tlsa(self):
         if app.config['TLS_FLAVOR'] in ('letsencrypt', 'mail-letsencrypt'):
             return [
                 # current ISRG Root X1 (RSA 4096, O = Internet Security Research Group, CN = ISRG Root X1) @20210902
-                f'_25._tcp.{hostname}. 86400 IN TLSA 2 1 1 0b9fa5a59eed715c26c1020c711b4f6ec42d58b0015e14337a39dad301c5afc3',
+                f'_25._tcp.{idna.encode(hostname.lower()).decode('ascii')}. 600 IN TLSA 2 1 1 0b9fa5a59eed715c26c1020c711b4f6ec42d58b0015e14337a39dad301c5afc3',
```

Comment:
5mins of caching for a TLSA pin?!? If anything, 1 day may arguably not be enough.